softwareconfiguration/mlplan/sklearn/conf/log4j.dtd

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Are we sure that we want the logging files in these commits?

Reply:
why not?